R/geojson_write.r

```diff
@@ -392,7 +394,7 @@ geojson_write.geo_list <- function(input, lat = NULL, lon = NULL, geometry = "po
   if (!overwrite && file.exists(file)) {
     stop(file, " already exists and overwrite = FALSE", call. = FALSE)
   }
-  cat(as.json(input, pretty = TRUE), file = file)
```

Comment:
Note that here I had to remove `pretty = TRUE` as default to allow passing `...`